verify_test.go

```diff
@@ -0,0 +1,88 @@
+package twilio
+
+import (
+	"context"
+	"net/url"
+	"testing"
+)
+
+func TestVerifyPhoneNumbersCreate(t *testing.T) {
+	t.Parallel()
+	client, s := getServer(verifyResponse)
+	defer s.Close()
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	data := url.Values{}
+	data.Add("to", "+14159373912")
+	data.Add("channel", "sms")
+	verify, err := client.Verify.VerifyPhoneNumbers.Create(ctx, "VA9e0bd45bfa7d9b9e7dca86cf94c7d4f8", data)
```

Comment:
Not a huge fan of the "Verify.Verify" stuttering here... maybe can we at least do "client.Verify.Verifications.Create" ?

Other thoughts on the API?

Reply:
I've renamed it.

The API looks good to me.